js/controllers/form/reviewer/ReviewerReviewStep3FormHandler.js

```diff
@@ -48,6 +50,29 @@
 	//
 	// Private methods.
 	//
+	/**
+	 * Internal callback called before form validation to ensure the
+	 * proper "required" state of the Recommendation field
+	 *
+	 * @param {HTMLElement} submitButton The submit button.
+	 * @param {Event} event The event that triggered the
+	 *  submit button.
+	 * @return {boolean} true.
+	 * @private
+	 */
+	$.pkp.controllers.form.reviewer.ReviewerReviewStep3FormHandler.
+			prototype.updateRecommendationRequired_ = function(submitButton, event) {
+
+		var $formElement = this.getHtmlElement(),
+				$recommendationElement = $formElement.find('[id^="recommendation"]');
+		if (submitButton.id.includes("submitFormButton-")) {
+			$recommendationElement.attr('required', '1');
+		} else {
+			$recommendationElement.removeAttr('required');
+		}
```

Comment:
Since this won't exist in OMP, I think that you'll need to check whether `$recommendationElement` contains a DOM element. You can use `if ($recommendationElement.length)`

Reply:
check